### ISSUE TITLE
Ordered list start value not copied over to pdfmake structure

### ISSUE THREAD

**Issue by rtaylorcengage** [Aymkdn/html-to-pdfmake#110]
An `<ol>`'s start value can specify that an ordered list should start from a certain number:
```html
<ol start="4">
    <li>Test list starting at item 4</li>
</ol>
```
html-to-pdfmake is currently ignoring this value when converting the content. The following output will render a list starting at 1, not 4:
```json
{
  "nodeName": "OL",
  "style": [
    "html-ol",
    "html-div"
  ],
  "ol": [
    {
      "text": "Test list starting at item 4",
      "nodeName": "LI",
      "style": [
        "html-li",
        "html-ol",
        "html-div"
      ]
    }
  ]
}
```
pdfmake itself does have the capacity to support the starting number if `start` is provided:
```json
{
  "nodeName": "OL",
  "start": 4,
  "style": [
    "html-ol",
    "html-div"
  ],
  "ol": [
    {
      "text": "Test list starting at item 4",
      "nodeName": "LI",
      "style": [
        "html-li",
        "html-ol",
        "html-div"
      ]
    }
  ]
}
```
The issue and proposed change has been confirmed in the html-to-pdfmake online converter and the pdfmake playground.

Could this please be added into html-to-pdfmake's functionality?

**Comment by Aymkdn** [comment]
I've just published version 2.3.1 that should support the `start` property.

**Comment by rtaylorcengage** [comment]
Thank you very much @Aymkdn for adding in this change so quickly!
